applications/nrf_desktop/README.rst

```diff
@@ -965,7 +965,7 @@ Sample mouse, keyboard or dongle (nrf52840dk_nrf52840)
       * The configuration uses the nRF52840 Development Kit.
       * The build types allow to build the application as mouse, keyboard or dongle.
       * Inputs are simulated based on the hardware button presses.
-      * The configuration with bootloader is available.
+      * The configuration with bootloader is set as default for sample mouse.
```

Comment:
Should we also add B0 to configuration working as dongle and keyboard? That would be consistent with release notes.
`Configurations without bootloader. By default all boards have B0 bootloader enabled if it fits the memory`

Reply:
What about configurations with shell ?

Reply:
Added